networkconfig/config.go

```diff
@@ -0,0 +1,20 @@
+package networkconfig
+
+import (
+	"fmt"
+
+	spectypes "github.com/bloxapp/ssv-spec/types"
+)
+
```

Comment:
@nkryuchkov 
this isn't important as part of this PR, but we'll eventually want to be able to add new network config on the fly. So a function that adds a config to `SupportedConfigs` (will probably require mutex).
then maybe load it from a json file for example. this will be useful for local testnet.
@Tomblox1 please create ticket.

Reply:
https://bloxxx.atlassian.net/browse/IO1-3440
@nkryuchkov @y0sher